spring-ai-model/src/main/java/org/springframework/ai/model/tool/DefaultToolCallingChatOptions.java

```diff
@@ -46,6 +47,9 @@ public class DefaultToolCallingChatOptions implements ToolCallingChatOptions {
 	@Nullable
 	private Boolean internalToolExecutionEnabled;
 
+	@Nullable
+	private Integer internalToolExecutionMaxIterations = ToolCallingChatOptions.DEFAULT_TOOL_EXECUTION_MAX_ITERATIONS;
```

Comment:
Consider moving the default value definition to this class rather than the interface, since this is the class implementing the defaults.